docs/installation/expansion.rst

```diff
@@ -132,12 +132,33 @@ following the template below:
        metalk8s.scality.com/ssh-key-path: /etc/metalk8s/pki/salt-bootstrap
        metalk8s.scality.com/ssh-host: <node control plane IP>
        metalk8s.scality.com/ssh-sudo: 'false'
+       metalk8s.scality.com/ssh-user: 'root'
      labels:
        metalk8s.scality.com/version: '|version|'
        <role labels>
    spec:
      taints: <taints>
 
+Annotations are used by Salt-SSH to connect to the node and deploy it.
+All annotations are prefixed with ``metalk8s.scality.com/``:
+
++--------------+--------------------------------------------------+-----------+
+| Annotation   | Description                                      | Default   |
++==============+==================================================+===========+
+| ssh-host     | Control plane IP of the node, must be accessible | None      |
```

Comment:
I think "accessible" in itself doesn't mean much, would be good to specify "accessible from the Bootstrap node"

Reply:
changed